src/main/webapp/app/code-editor/file-browser/code-editor-file-browser.component.ts

```diff
@@ -114,6 +117,7 @@ export class CodeEditorFileBrowserComponent implements OnInit, OnChanges, AfterV
             }
             this.gitConflictState = gitConflictState;
         });
+        this.guidedTourService.enableTour(codeEditorTour);
```

Comment:
Imo the tour should be enabled in the code-editor-student-container.component.

Reply:
Thank you for the suggestion - that makes more sense :)